frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -390,6 +426,36 @@ export default ({
         bottom: `${innerHeight - bbox.top + 8}px` // 8 -> 0.5rem gap
       })
     },
+    openFileAttach (e) {
+      e.target.blur()
+      this.$refs.fileAttachmentInputEl.click()
+    },
+    fileAttachmentHandler (filesList) {
+      const targetFile = filesList[0]
+      const fileUrl = URL.createObjectURL(targetFile)
+      const fileSize = targetFile.size
+
+      if (fileSize > Math.pow(10, 9)) {
```

Comment:
`Math.pow(10, 9)` here should be replaced with a constant whose name tells the purpose such as `FILE_ATTACHMENT_SIZE_LIMIT`. will make the change along with other upcoming change requests.

Reply:
FYI, this value will come from the server via some server API, as the server dictates the maximum file size people can upload

Reply:
@taoeffect I will leave a comment saying that is a future TODO change once the backend for it is implemented.